test/indent.txt

```diff
@@ -19,12 +19,74 @@ to go
 ++]
 end
 
+IfMultiBracket1
+to foo
+  let abc 0
+  if true [ set abc [
+    1 2 3 ]
+  ] [
+++  set abc [4 5 6]
+  ]
+end
+
+# I'm not certain this should be normative
+# This behavior wasn't specified prior to 6.0.1 and could probably change
+IfMultiBracket2
```

Comment:
Yeah, thanks for the feedback on this. I ended up breaking it down into several cases and discovered an even better indentation strategy where it looks through the indentation stack for the last bracket matched instead of the delta between openers and closers.